src/components/FileExplorer.vue

```diff
@@ -152,11 +158,55 @@ function onNodeDoubleClicked(node) {
   FileEvent.SelectFileNodeEvent.next(node);
 }
 
+/**
+ * Open parsable files contained in the given parent node.
+ * @param {Object} node - QTree parent node.
+ */
+function openParsableFiles(node) {
+  const plugin = getPluginByName(query.value.path.split('/')[0]);
+
+  node.children.forEach((child) => {
+    if (plugin.isParsable({ path: child.id })) {
+      onNodeDoubleClicked(child);
+    }
+  });
+}
+
+/**
+ * Open folders recursively.
+ * @param {Array} folders - Array of folder keys.
+ */
+function expandFolders(folders) {
+  const node = fileExplorerRef.value.getNodeByKey(folders[0]);
+
+  if (node) {
+    fileExplorerRef.value.setExpanded(folders[0], true);
+
+    if (folders.length > 1) {
+      expandFolders(folders.slice(1));
+    } else {
+      openParsableFiles(node);
+    }
+  }
+}
+
 /**
  * Update file explorer tree nodes.
+ * Also, expand model folder and open parsable files.
  */
 function updateFileExplorer() {
   nodes.value = getTree(props.projectName, props.fileInformations);
+
+  // Didn't a proper way to stub it on shallowMount.
```

Comment:
`Didn't find a proper way...`
Also why specify this here ?

Reply:
because the if under is only to pass the test. To Delete if another solution is found.

Reply:
done